### DISCUSSION TITLE
Modeling a waypoint navigation system - UEReferenceApp

### DISCUSSION THREAD

**Discussion by MariaALJ** [JSBSim-Team/jsbsim#747] [Q&A] [answered]
Hello, I'm interested in a case study from the JSBSim reference manual which is about modeling a waypoint navigation system (section4, case 5). My aim is to adapt this model to the Unreal Engine ReferenceApp which im using to learn about JSBSim. Im very new to this field (i mentioned it in another discussion) so I wanted to ask: is there anyone that could help me with the steps that i have to follow in order to be able to implement this model in the UEReference App? Thank you very much!

**Comment by gallonmate** [comment, marked as answer]
Script xml files currently are not supported in the plugin. Only loading the Aircraft xml files is possible right now. The Unreal Engine plugin is not feature complete and is not 100% integrated to JSBSim, if you want feature complete you can try FlightGear.

That being said you can try to load an aircraft that has Auto Pilot, like the c172x and then in Unreal Engine Blueprints use the JSBSim Command Node to manually enable and set the different auto pilot commands. But this is also untested so some of the commands may not work in Unreal Engine.

**Reply by MariaALJ** [reply]
Thank you very much for your help!
